config.yml

```diff
@@ -35,15 +35,15 @@ pipeline:
 - name: FallbackClassifier
   threshold: 0.7
 policies:
+- name: RulePolicy
+  core_fallback_threshold: 0.3
+  core_fallback_action_name: "action_default_fallback"
+  enable_fallback_prediction: True
+- max_history: 6
+  name: AugmentedMemoizationPolicy
 - name: TEDPolicy
   max_history: 10
   epochs: 20
   batch_size:
   - 32
   - 64
```

Comment:
changed order of policies so that if there are rule conflicts they will be determined before ted policy trains to save time